src/materials/ConcreteThermalMoisture.C

```diff
@@ -386,8 +386,6 @@ ConcreteThermalMoisture::computeProperties()
               _thermal_conductivity[qp] = 1.4875;
             else if (T >= 20.0 && T < 800.0)
```

Comment:
You probably don't need the T<800 check here, right?

Reply:
I have removed it now.